src/modules/launcher/Plugins/Microsoft.Plugin.Calculator/Main.cs

```diff
@@ -88,8 +88,11 @@ public List<Result> Query(Query query)
                         } 
                     };
                 }
-            }
-            catch
+            }
+//We want to keep the process alive if any the mages library throws any exceptions.
+#pragma warning disable CA1031 // Do not catch general exception types
+            catch
+#pragma warning restore CA1031 // Do not catch general exception types

```

Comment:
Would it be better to do this or to actually log the exception? 
I can't think of a scenario where we would actually be hitting this so wouldn't it be better to log that unexpected case?

Reply:
fixed.